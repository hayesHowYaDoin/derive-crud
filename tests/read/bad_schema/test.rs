use derive_crud::Read;

#[derive(Read)]
#[crud_table("test_table")]
struct Test {
    #[crud_id]
    id: i64,
    name: String,
    location: String,
}

#[tokio::main]
async fn main() {
    let _result = Test::read(1).await.unwrap(); // Test should fail before unwrap
}
