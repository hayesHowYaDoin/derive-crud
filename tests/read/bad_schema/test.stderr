error[E0063]: missing field `location` in initializer of `Test`
 --> tests/read/bad_schema/test.rs:5:8
  |
5 | struct Test {
  |        ^^^^ missing `location`
